### DISCUSSION TITLE
how to use NetCDF data to generate wind flow field?

### DISCUSSION THREAD

**Discussion by bizarre-hhh** [pelicanmapping/osgearth#2654] [Support Q&A]
how to use NetCDF data to generate wind flow field?  version osgearth 3.2.
like this pic and gif
![wind](https://github.com/user-attachments/assets/b0a61614-20e9-4fe5-80e0-ca10283ab133)
![wind2](https://github.com/user-attachments/assets/a8b9e0aa-8426-4ec8-8845-89137427739e)

can windLayer to do ?
![image](https://github.com/user-attachments/assets/079fdcf1-c253-4a85-a8d2-d625b02c0a82)

**Comment by gwaldron** [comment]
WindLayer will not help you.

There is an old (2020) demo that does this sort of thing. You will need to adapt it to your particular environment.
Good luck!

https://github.com/gwaldron/osgearth/blob/wind_particles/src/applications/osgearth_wind/osgearth_wind.cpp
